.babelrc

```diff
@@ -18,7 +18,7 @@
   ],
   "env": {
     "development": {
-      "plugins": ["react-hot-loader/babel"]
+      "plugins": ["react-hot-loader/babel", "react-refresh/babel"]
```

Comment:
Ideally we would apply the Babel plugin depending on which HMR option is enabled. Now both are enabled during development.

Reply:
it's okay if it doesn't matter much :)

Reply:
Ok, the first one can be dropped when everything is on RFR.